### PR TITLE
docs(README): cleanup ecosystem table

### PR DESCRIPTION
**Pull request by irmerk**
# No Issue

### Changes
- Change PR template to point to closing an issue
- Remove all contributors to `README`
- Update `markdown-transform` ecosystem

### Flags
- N/A

### CODE REVIEW

**Comment by irmerk** [comment]
See preview [here](https://github.com/irmerk/techdocs/tree/irmerk/cleanup)

File: README.md [hunk 1]
```diff
@@ -212,17 +251,12 @@ Accord Project is an open source, non-profit, initiative working to transform co
   </tr>
   <tr>
     <td headers><a href="https://github.com/accordproject/template-studio">Template Studio v1</a></td>
```
Comment (jeromesimeon) [inline README.md:253]: Remove v1?

File: README.md [hunk 2]
```diff
@@ -212,17 +251,12 @@ Accord Project is an open source, non-profit, initiative working to transform co
...
-    <td headers> <b>0.13.4</b></td>
-    <td headers>Web UI for creating, editing and testing Accord Project templates</td>
-  </tr>
-  <tr>
-    <td headers><a href="https://github.com/accordproject/template-studio-v2">Template Studio v2</a></td>
-    <td headers> <b>0.13.4</b></td>
+    <td headers> <b>0.21.0</b></td>
```
Comment (jeromesimeon) [inline README.md:254]: Remove version altogether?

**Review by jeromesimeon — APPROVED** [review]
Looks great. Just couple of (optional) suggestions.

**Pull request merged**